File: tugasKelompok/Matematika 2 (Teorema Pythagoras)/C++/teorema_pythagoras.cpp
#include <stdio.h>
#include <math.h>
int main(){
    int a, b;
    double sisiMiring;
    printf("\tmenghtung sisi miring segitiga siku-siku");
    printf("Input panjang sisi a "); scanf("%d", &a);
    printf("Input panjang sisi b "); scanf("%d", &b);
    sisiMiring = sqrt(pow(a, 2)+pow(b, 2));
    printf("%.2f", sisiMiring);
    return 0;
}

File: tugasKelompok/Matematika 2 (Teorema Pythagoras)/Pascal/teorema_pythagoras.pas
program teoremaPythagoras;
var a, b: integer;
sisiMiring: real;

begin
writeln('menghtung sisi miring segitiga siku-siku');
write('Masukan panjang sisi a ');readln(a);
write('Masukan panjang sisi b ');readln(b);
sisiMiring :=sqrt(sqr(a)+sqr(b));
writeln('Panjang sisi miring segitiga siku-siku dengan panjang sisi a ', a, ' dan panjang sisi b ', b, ' adalah ',sisiMiring:3:2);
end.